sway/commands/focus.c

```diff
@@ -194,6 +194,9 @@ static struct cmd_results *focus_output(struct sway_seat *seat,
 	}
 	char *identifier = join_args(argv, argc);
 	struct sway_output *output = output_by_name(identifier);
+	if (!output) {
+		output = output_by_identifier(identifier);
```

Comment:
Maybe we could replace `output_by_{name,identifier}` by a single function? Don't know if we use one without the other somewhere.